src/index.html

```diff
@@ -18,5 +31,81 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="images"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="Apple"
```

Comment:
The `href` attribute should contain a valid URL or path. Currently, it is set to 'Apple', which is not a valid URL. Consider using a relative path like './apple.html' or an absolute URL.